actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -419,6 +419,12 @@ def protect_against_forgery? # :doc:
       def valid_request_origin? # :doc:
         if forgery_protection_origin_check
           # We accept blank origin headers because some user agents don't send it.
+          raise InvalidAuthenticityToken, "The browser returned a 'null' origin for a request with origin-based" \
+          " forgery protection turned on. This usually means you have the 'no-referrer' Referrer-Policy header " \
+          "enabled, or that you the request came from a site that refused to give its origin. This makes it " \
+          "impossible for Rails to verify the source of the requests. If you are sure that this is the "\
+          "behaviour you want, you can disable origin checking by doing "\
+          "Rails.application.config.action_controller.forgery_protection_origin_check = false" if request.origin == "null"
```

Comment:
Do you think most people will think to look for a postfix conditional at the end of a 6 line expression?